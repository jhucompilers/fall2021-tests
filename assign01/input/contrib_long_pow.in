9007199254740993 ^ 1;