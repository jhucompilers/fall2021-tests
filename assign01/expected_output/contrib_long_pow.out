Result: 9007199254740993
